src/lib.rs

```diff
@@ -490,4 +490,18 @@ impl WasccHost {
         let lock = self.caps.read().unwrap();
         lock.clone()
     }
+
+    pub fn actors_by_tag(&self, tags: &[&str]) -> Vec<String> {
+        let mut actors = vec![];
+
+        for (actor, claims) in self.claims.read().unwrap().iter() {
+            if let Some(actor_tags) = claims.metadata.as_ref().and_then(|m| m.tags.as_ref()) {
+                if tags.iter().all(|&t| actor_tags.contains(&t.to_string())) {
```

Comment:
I haven't run this yet but, it looks like this could potentially insert the same actor multiple times if more than one of their tags matches the search criteria?